### DISCUSSION TITLE
Destination folder

### DISCUSSION THREAD

**Discussion by RustyHorns** [shish/shimmie2#1135] [Q&A] [answered]
Hello, sorry if this isn't presented correctly, I don't use Github much. 
Is there a way to change the folder where Shimmie2 save the pictures? For context I installed it on an SDD, thinking I could make it save the pictures on a larger HDD while having the program itself on the SSD to load it faster at startup, but I don't have much space left on the SSD.
If there is no way to do that I can just reinstall it on the HDD, but if possible I'd rather not have to fumble with the files by moving everything around or reinstall it completely.

**Comment by shish** [comment, marked as answer]
This depends more on your setup than on shimmie itself - in particular windows / linux / mac as a server, and are you running via docker, and what web server

As vague pointers - for any kind of linux or mac server, it should be possible to move the "data" folder to a new place and then create a symlink from the old to the new place (I think modern windows has something equivalent but I forget the name); alternatively since everything is calculated relative to where shimmie is installed, you can move the entire install folder safely without needing to reinstall

**Reply by RustyHorns** [reply]
Oh I see, thank you very much! I'm a beginner at this and didn't want to fiddle with things too much in fear of messing it up, I'll be trying that!

Have a nice day.
